### PR TITLE
SEO-173296 WPF Chart Header

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, and changed image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Header.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ chart.Header = "Usage of Metals";
 
 {% endtabs %}
 
-![WPF Chart with Header](Area_images/wpf-chart-header.jpeg)
+![WPF Chart with Header](area_images/wpf-chart-header.jpeg)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Header.md:32]: Changed this name in folder

Comment (Yvone-Atieno) [reply wpf/Charts/Header.md:32]: the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**